Tests/PackageLoadingTests/PD5LoadingTests.swift

```diff
@@ -496,4 +496,64 @@ class PackageDescription5LoadingTests: PackageDescriptionLoadingTests {
             XCTAssertMatch(message, .contains("was introduced in PackageDescription 5.2"))
         }
     }
+
+    // TODO: Remove the underscore when we have fixed what tools version binary targets will appear in.
+    func _testBinaryTargetUnavailable() throws {
+        do {
+            let stream = BufferedOutputByteStream()
+            stream <<< """
+                import PackageDescription
+                let package = Package(
+                    name: "Foo",
+                    products: [],
+                    targets: [
+                        .binaryTarget(
```

Comment:
shouldn't this be _binaryTarget?

Reply:
Correct! But mind you, those tests are disabled for now.